src/scripts/main.js

```diff
@@ -21,7 +21,34 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key1 in food) {
```

Comment:
The `for...in` loop should iterate over `data` instead of `food` to utilize the `data` parameter passed to the `createTree` function.